docs/concepts/who.qmd

```diff
@@ -0,0 +1,59 @@
+# Who supports Ibis?
+
+Ibis is an open-source project that welcomes contributions from anyone! We have
+a growing community of users and contributors, and we'd love to have you join
+us. If you're interested in contributing, please see our [contributing
+guide](/contribute).
+
+## Voltron Data
+
+[Voltron Data](https://voltrondata.com) is the primary sponsor of Ibis, with
+most of the core development team employed there. As of writing, this includes
+five full-time developers, one technical product manager, and other staff who
+contribute to Ibis.
+
+## Other companies
+
+Ibis is used by many other companies, with various tools built on top of it.
+These include:
+
+- [Google BigQuery DataFrames](https://github.com/googleapis/python-bigquery-dataframes), a clone of the pandas API built on Ibis
+- [Starburst Galaxy Python DataFrames](https://www.starburst.io/blog/introducing-python-dataframes/), with support for Ibis
+- [Claypot AI's contribution of the Flink backend](https://github.com/claypotai/ibis-flink-example), working in collaboration with Voltron Data
+- [Microsoft's Magpie project](https://www.microsoft.com/en-us/research/project/magpie-2/), built on top of Ibis
+- [SuperDuperDB](https://github.com/SuperDuperDB/superduperdb), bringing AI to any backend Ibis supports
+
+## History
+
+Ibis was originally created by [Wes McKinney](https://wesmckinney.com/). Wes
+created pandas, co-created Apache Arrrow, and co-founded Voltron Data (among
+other things). Ibis was initially a pandas-like dataframe library for Apache
+Impala, but has since grown to support many other backends and mature under the
+stewardship of [Phillip Cloud](https://github.com/cpcloud) and others on the
```

Comment:
@cpcloud would you want this linking somewhere else? I don't think you have a personal website and linking to youtube seemed weird

Reply:
GitHub is fine, thanks!